graphviz/lang.py

```diff
@@ -50,10 +50,23 @@ def quote(identifier,
     if html(identifier) and not isinstance(identifier, NoHtml):
         pass
     elif not valid_id(identifier) or identifier.lower() in dot_keywords:
-        return '"%s"' % identifier.replace('"', '\\"')
+        return '"%s"' % _escape_id_characters(identifier)
```

Comment:
IIUC, it should be enough to change this to `identifier.replace('\\', '\\\\').replace('"', '\\"')`

Reply:
I think you're right. If you do slash before the quote, it should be fine.

Good catch. That is more efficient and cleaner.